requirements.txt

```diff
@@ -18,3 +18,4 @@ psutil
 gunicorn
 requests
 pandas
+pymedphys==0.34.0-dev12
```

Comment:
0.35.0 is released now 🙂 https://pypi.org/project/pymedphys/0.35.0/

Should be able to use that instead.

Reply:
Will do, thanks!